docker-compose.yml

```diff
@@ -2,10 +2,26 @@ version: '3.7'
 
 services:
 
+  mysql:
+    image: mysql:8
+    restart: always
+    environment:
+      MYSQL_DATABASE: 'dyescape-bot'
+      MYSQL_USER: 'dyescape'
+      MYSQL_PASSWORD: 'bot'
+      MYSQL_ROOT_PASSWORD: 'dyescape-bot'
+    ports:
+      - '3306:3306'
```

Comment:
Sure you want these ports open on the host machine? (If not -> expose)

Reply:
No no no, the docker-compose files are 100% purely there only for development and local environments. All production deployments will be done through Kubernetes and indeed those will not expose anything, especially not database connectivity, to the public. Passwords will also be configured on the fly through secret environment variables.